google/cloud/aiplatform/helpers/__init__.py

```diff
@@ -1,3 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 from google.cloud.aiplatform.helpers import value_converter
+from google.cloud.aiplatform.helpers.container_uri_builders import (
+    get_prediction_container_uri,
+)
 
-__all__ = (value_converter,)
+__all__ = ("get_prediction_container_uri" "value_converter",)
```

Comment:
```suggestion
__all__ = ("get_prediction_container_uri", "value_converter",)
```


I'm uncertain of exposing `value_converter` as well as it's not a module we currently expose or provide documentation/usage on. It mainly used by the enhanced portions of the library.

Reply:
Could we potentially move `value_converter` into `aiplatform.utils` so that `aiplatform.helpers` contains only methods intended for external use?

Reply:
SGTM